sites/default/menu_data.json

```diff
@@ -502,13 +502,6 @@
             "children" : [ ],
             "requirement" : 0
         }, { 
-            "label" : "Referrals",
```

Comment:
Yep It got back in. I have the report and was going to look at it and see what I could do to fix it. I removed this so there was no broken link